bin/game.rb

```diff
@@ -0,0 +1,103 @@
+require_relative '../lib/board.rb'
+
+class Game
+
+    def initialize
+        @board = Board.new()        
+        @moves = 0
+    end  
+
+    def show()
+        puts ""
+        puts " #{@board.get(1)} | #{@board.get(2)} | #{@board.get(3)} "
+        puts "---|---|---"
+        puts " #{@board.get(4)} | #{@board.get(5)} | #{@board.get(6)} "
+        puts "---|---|---"      
+        puts " #{@board.get(7)} | #{@board.get(8)} | #{@board.get(9)} "
+    end
+    
+    def validatePosition(position)
+        #7: non stopping error
+        #0: everything OK
+        unless position.is_a? Integer
+            puts "Position is not an integer"
+            return 7
+        end
+        if position > 9
```

Comment:
Position cannot be more than 9 and less than 1 at same time so this could be an if else
```ruby
if position > 9
  return # whatever you got to return
else 
  return # whatever you need to return
end
```

could even be it's own private method and based on that return in another method a validated message
In fact, yo ucan have that method and another method that returns a message that can be used in gameplay script